### ISSUE TITLE
[Profiles] How do I delete these folders and workspaces?

### ISSUE THREAD

**Issue by kands-code** [microsoft/vscode#230578]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 
  ```plaintext
  > code --version
  1.94.0
  d78a74bcdfad14d5d3b1b782f87255d802b57511
  x64
  ```
- OS Version: ArchLinux and macOS 15.1

I found that this "Folders & Workspaces" is not configurable in my profile, or that I can only add folders, but not remove them.

![截图 2024-10-05 18-34-59](https://github.com/user-attachments/assets/24fe4033-3728-4572-b7ad-097ac3f8deae)

Many of the folders in this picture have actually been deleted or changed location, but they are still there and there is no obvious way to remove them, other than copying this profile and then deleting the original profile, which I think is a design failing as this content should be local to the user and should not be synchronised, or at least there should be a way to remove them.

Related Links: [https://code.visualstudio.com/docs/editor/profiles#_folder-workspace-associations](https://code.visualstudio.com/docs/editor/profiles#_folder-workspace-associations)

**Comment by tk3dots** [comment]
I'd also like to know how to delete them.

**Comment by russeg** [comment]
i don't mind deleted locations are still there, if they are automatically cleanup up. those entries could grow very long over time.

**Comment by kands-code** [comment]
> i don't mind deleted locations are still there, if they are automatically cleanup up. those entries could grow very long over time.
> 

The fact is that these content can not be cleaned up now, let alone "automatically". 😞

**Comment by sandy081** [comment]
/duplicate

This will be supported in the next version. Thanks.

**Comment by kands-code** [comment]
> This will be supported in the next version. Thanks.

VSCode 1.94.1 still doesn't fix this. What is the "next version"?

**Comment by sandy081** [comment]
It will be available in 1.95.0
